docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source.md

```diff
@@ -0,0 +1,197 @@
+---
+id: google-bigquery-source
+title: Google BigQuery Source
+sidebar_label: Google BigQuery
+description: Learn how to collect data using the BigQuery API.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/google-bigquery-icon.png')} alt="google-bigquery-icon" width="70" />
```

Comment:
Can you confirm this matches the UI image? The one we use for BigQuery app is different https://help.sumologic.com/docs/integrations/google/bigquery

Reply:
@kimsauce This is the icon from our stag environment. Previously we have Gmail BigQuery and this is Google BigQuery and both have different uses.

Reply:
oh i see, thanks for confirming!